x/qgb/types/query.pb.go

```diff
@@ -6,8 +6,8 @@ package types
 import (
 	context "context"
 	fmt "fmt"
-	_ "github.com/cosmos/cosmos-proto"
 	types "github.com/cosmos/cosmos-sdk/codec/types"
+	_ "github.com/cosmos/cosmos-proto"
```

Comment:
This one is expected i guess. Could you please gofumpt and see if this change is reverted?